test/elm/string/data.cql

```diff
@@ -58,3 +58,19 @@ define TooMuchLength: Substring('HelloWorld', 7, 25)
 define NegativeLength: Substring('HelloWorld', 7, -1)
 define NullString: Substring(null, 5)
 define NullStart: Substring('HelloWorld', null)
+
+// @Test: StartsWith
+define FooBarStartsWithFoo: StartsWith('FooBar', 'Foo')
+define FooBarStartsWithBar: StartsWith('FooBar', 'Bar')
+define FooBarStartsWithBlank: StartsWith('FooBar', '')
+define BlankStartsWithFoo: StartsWith('', 'Foo')
+define StartsWithNull: StartsWith('FooBar', null as String)
```

Comment:
Should you also add tests that just passes in `null` without the `as String`?  I am not entirely sure what the `as String` does.

Reply:
The `as String` is just a mechanism for hinting to the cql-to-elm translator what the intended type of the `null` is.  Without it, sometimes the translator cannot properly resolve signatures.

I'll try adding a test without the `as String`, but it may fail the compiler -- I'm not sure.

Reply:
New tests pushed.